app/javascript/article-form/components/__tests__/Form.test.jsx

```diff
@@ -6,13 +6,29 @@ import { Form } from '../Form';
 let bodyMarkdown;
 let mainImage;
 
+// Axe flags an error for the multi-line combobox we use for Autosuggest, since a combobox should be a single line input.
+// This is a known issue documented on #13044
+const customAxeRules = {
```

Comment:
The custom axe rules are only required for tests related to the mention auto complete right?

Reply:
Yep they are just for these specific two tests below. I've added to the comment to make that clear though